### PR TITLE
Milestone: Project Setup

### PR DESCRIPTION
**Pull request by Kashif-Saleem-Ghuman**
### In this milestone:

- [ ] I have setup the project using create react app
- [ ] I have set up the linters
- [ ] I have created the components for books and categories
- [ ] I have used routes to navigate to the pages 
- [ ] I have done basic styling

### CODE REVIEW

**Review by emmiiorji — COMMENTED** [review]
Hi @Kashif-Saleem-Ghuman, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Invalid

You have submitted a pull request without a description. Kindly provide a proper description of the  pull request telling the changes implemented.
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Kashif-Saleem-Ghuman** [comment]
@emmiiorji  Thankyou for the feedback. I completely forgot to add it.

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,30 @@
...
-    </div>
+    <>
+      <nav className="Nav-bar">
+        <h1>
+          Bookstore CMS
+        </h1>
+        <ul className="ul">
+          <li>
+            <Link className="li-app" to="/books"> Books</Link>
+          </li>
```
Comment (HAMMAS-SALEEM) [inline src/App.js]: - [ ] Your books page is the homepage or route page of your application means it should not be on a specific URL. Kindly check **to="/books"** to **to="/"** to make it route component :blush:

File: src/Components/Books.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      <ul className="ul-books">
+        <li className="li-books">
+          <span>{title}</span>
+          <span>{author}</span>
+          <button type="submit">Remove</button>
+        </li>
+      </ul>
+    </>
+  );
+}
```
Comment (HAMMAS-SALEEM) [inline src/Components/Books.js]: project

- [ ] Kindly see that in your design there is a form to **Add New Book** but you don't have it here :fearful: Kindly add the form to follow the requirements :+1:

**Note: Styling is not required at this point :+1:**

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @Kashif-Saleem-Ghuman :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kashif-Saleem-Ghuman** [comment]
@HAMMAS-SALEEM  Thank you for reviewing my PR and providing valuable feedback. I will implement the changes and submit it again.

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @Kashif-Saleem-Ghuman, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Kashif-Saleem-Ghuman** [comment]
@emmiiorji Thank you for such wonderful comments and valuable feedback regarding the commit messages. I will definitely keep that in mind.

**Pull request merged**